sample/src/main/res/layout/activity_events.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright 2014 Alex Curran
```

Comment:
2015? :tongue: 
I never understood if you should use the creation date of the file or of the library, in this kind of comments..